data/campaigns/Heir_To_The_Throne/scenarios/19c_Cliffs_of_Thoria.cfg

```
@@ -767,24 +773,34 @@ Soooo... It is you who sent your subordinates to attack us. Now when we’ve des
         [/message]
         [message]
             speaker="Li'sar"
-            message= _ "Whatever... I still think we should make an effort to rescue him from the unfortunate position he is in. If you help him, I’ll do my best to convince him that you are not his enemies but his friends..."
-        [/message]
-        [message]
-            speaker=Konrad
-            message= _ "Our main goal remains to survive ourselves, but if we can spare the troops, we might try to rescue the sergeant stranded on the cliff. However, I make no promises."
+            message= _ "Be that as it may, I still think we should make an effort to rescue him from the unfortunate position he is in. If you help him, I’ll do my best to convince him that you are not his enemies but his friends."
         [/message]
         [message]
             speaker=Kalenz
-            message= _ "Fair enough!"
+            message= _ "Very well. If you convince him not to raise his sword against us, the Lintair elves shall grant him safe passage."
         [/message]
         [message]
-            speaker="Li'sar"
-            message= _ "Fair enough."
+            speaker=Konrad
+            message= _ "Our main goal remains to survive ourselves, but if we can spare the troops, we might try to rescue the sergeant stranded on the cliff. However, I make no promises."
         [/message]
         [message]
-            speaker=Delfador
-            message= _ "Fair enough..."
-        [/message]
+            speaker="Li'sar"
+            message= _ "That I do understand, Konrad. I am no stranger to the burden of command."
+        [/message]
+        {IF_VAR waterfall_sighted boolean_equals yes (
+            [then]
+                [message]
+                    speaker=Konrad
+                    message= _ "I’m glad we agree, Princess. Now, onward! We must reach the cave."
```

```suggestion
                    message= _ "I’m glad we agree, Li'sar. Now, onward! We must reach the cave."
```

We removed "Princess" as a direct address before, so better to not reintroduce it.

---

I'd accept the suggestion, but it uses an ASCII quote where a typographical one is needed.

---

Fixed.